paddle/fluid/framework/ir/graph_pattern_detector.cc

```diff
@@ -840,6 +840,46 @@ PDNode *patterns::ElewiseAddActInplaceGrad::operator()(
   return ele_add_grad;
 }
 
+PDNode *patterns::Conv::operator()() {
+  auto conv_op = pattern->NewNode(conv_op_repr())->assert_is_op("conv2d");
+
+  auto input_var = pattern->NewNode(conv_input_repr())
+                       ->assert_is_op_input("conv2d", "Input");
+
+  auto bias_var =
+      pattern->NewNode(conv_bias_repr())->assert_is_op_input("conv2d", "Bias");
+
+  auto filter_var = pattern->NewNode(conv_filter_repr())
+                        ->assert_is_op_input("conv2d", "Filter");
+
+  auto output_var = pattern->NewNode(conv_output_repr())
+                        ->assert_is_op_output("conv2d", "Output");
+
+  conv_op->LinksFrom({input_var, bias_var, filter_var});
+  conv_op->LinksTo({output_var});
+
+  return output_var;
+}
+
+PDNode *patterns::ElementwiseAdd::operator()(PDNode *conv_output) {
```

Comment:
Could this pattern also be found after other nodes than convolution? If so, maybe a more generic name than conv_output would be appropriate?

Reply:
On the other hand, when I do conv-eltwise_add-bn I use Y for bias and x for convolution output and you do it as the opposite. I wonder if that matters? If this elementwise_add would be re-used in conv-elementwise_add-bn then I could just switch x and y in the fuse code and it should still work?